sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -58,6 +58,10 @@ object ScalaJSPlugin extends AutoPlugin {
     val scalaJSIsSnapshotVersion = ScalaJSVersions.currentIsSnapshot
     val scalaJSBinaryVersion = ScalaJSCrossVersion.currentBinaryVersion
 
+    object ScalaJSTags {
+      val Link = Tags.Tag("scalajs-link")
```

Comment:
I'd like to see some documentation on this. Basically a description that it is a tag on every task that links to allow restricting concurrency.

Reply:
I have added some docs now, please let me know if these look okay